test/resources/report-schemas.test.ts

```diff
@@ -72,7 +74,44 @@ it("schema of facial similarity photo report should be valid", async () => {
     "complete"
   );
 
+  const extraProperties = {
+    id_photos: [],
+    live_photos: [],
+    live_video: [],
+    motion_captures: []
```

Comment:
We've spotted those four new fields appearing in DocumentReport, they need to be added to openapi spec if they're confirmed to stay.

Reply:
What has been spotted here is going to be addressed in: https://github.com/onfido/onfido-openapi-spec/pull/133